mycroft/listener/mic.py

```diff
@@ -42,17 +46,20 @@
     play_wav, play_ogg, play_mp3
 )
 from mycroft.util.log import LOG
-from mycroft.deprecated.speech_client import NoiseTracker
-from mycroft.listener.data_structures import RollingMean, CyclicAudioBuffer
-from mycroft.listener.silence import SilenceDetector, SilenceResultType, SilenceMethod
-
+from ovos_utils.configuration import get_xdg_data_save_path
 from ovos_plugin_manager.vad import OVOSVADFactory
-
+from ovos_utils.messagebus import get_message_lang
 
 WakeWordData = namedtuple('WakeWordData',
                           ['audio', 'found', 'stopped', 'end_audio'])
 
 
+class ListeningMode(str, Enum):
```

Comment:
Is this a mode or a state? It seems like this is used more to track state in the listener. Consider adding a docstring or comments to define what each mode means